### PR TITLE
docs(readme): add description for markdown support

### PR DESCRIPTION
**Pull request by ikatyang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -838,7 +840,7 @@ features enabled, but you can also use the
 All of JSX and Flow syntax is supported. In fact, the test suite in
 `tests/flow` *is* the entire Flow test suite and they all pass.
 
-Prettier also supports [TypeScript](https://www.typescriptlang.org/), CSS, [Less](http://lesscss.org/), [SCSS](http://sass-lang.com), [JSON](http://json.org/), and [GraphQL](http://graphql.org/).
+Prettier also supports [TypeScript](https://www.typescriptlang.org/), CSS, [Less](http://lesscss.org/), [SCSS](http://sass-lang.com), [JSON](http://json.org/), [GraphQL](http://graphql.org/), and [Markdown](http://commonmark.org/).
```
Comment (lipis) [inline README.md]: I think we don't need comma (`,`) before **and**.

Comment (hawkrives) [reply README.md]: I mean, that's the whole "oxford comma" debate, isn't it? 

(Disclaimer: I love the oxford comma.)

Given that the file had the comma before, I'd think that it ought to still have it. 😃

Comment (ikatyang) [reply README.md]: Not sure which one is better, remain the same style as before.

Comment (azz) [reply README.md]: I am pro Oxford and trailing comma :wink:

**Pull request closed without merging**